src/sentry/grouping/fingerprinting/configs/sentry.javascript@2024-02-02/002_hydration_errors.txt

```diff
@@ -0,0 +1,5 @@
+family:javascript tags.transaction:"*" message:"Hydration failed because the initial UI does not match what was rendered on the server." -> hydrationerror, {{tags.transaction}}
```

Comment:
Is `tags.transaction:"*"` necessary?
Are you trying to check if the tag transaction is set? (e.g. `has:transaction`)

Reply:
I can not find it in the docs but I know that it works for certain fields/tags in Discover (which is the same syntax we use for fingerprint).

Here are some examples:
<img width="719" alt="image" src="https://github.com/getsentry/sentry/assets/44410/11c96e5d-a07d-4caa-bbb7-60a01fd8ef14">
<hr />
<img width="715" alt="image" src="https://github.com/getsentry/sentry/assets/44410/9c653417-9710-4551-ac11-bf9a8579760d">
<hr />
<img width="716" alt="image" src="https://github.com/getsentry/sentry/assets/44410/d76e8458-a516-4ad1-a4b7-6d702722997e">

Reply:
Interesting. For what it's worth this `tags.transaction:"*"` is in [the current master](https://github.com/getsentry/sentry/blob/6f3dd03c8297e77592fe0ce4112895513ba6c3f7/src/sentry/grouping/fingerprinting/configs/sentry.javascript.nextjs%402023-12-22/002_hydration_errors.txt). This only appears like an new file, because `git mv` is useless. So I'd rather change that in a separate PR.

Reply:
No need to change anything if this works for you. I don't understand the differences. Your syntax usage is just new to me.

Reply:
`has:transaction` better shows the intention, so I'll change that, but in a separate PR.